ai-gateway/app/main.py

```
@@ -0,0 +1,78 @@
+                    "prompt": request.prompt,
+                    "stream": False
+                },
+                timeout=60.0
+            )
+            response.raise_for_status()
+            return response.json()
+        except httpx.RequestError as e:
+            raise HTTPException(status_code=503, detail=f"Ollama service unavailable: {str(e)}")
+
+@app.post("/api/v1/ingest/document")
+async def ingest_document(file: UploadFile = File(...)):
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)